workflow/metrics/metrics.go

```diff
@@ -117,3 +120,18 @@ func (m *Metrics) populate(ctx context.Context, adders ...addMetric) error {
 	}
 	return nil
 }
+
+func getTemporality(config *Config) metricsdk.TemporalitySelector {
+	switch config.Temporality {
+	case wfconfig.MetricsTemporalityCumulative:
+		return func(metricsdk.InstrumentKind) metricdata.Temporality {
+			return metricdata.CumulativeTemporality
+		}
```

Comment:
is there no `metricdata.CumulativeTemporalitySelector`? since `Default` exists I would think the other two would too

Reply:
I'd also have thought this, but no.